### DISCUSSION TITLE
Not showing interactive widgets on readthedocs built webpage

### DISCUSSION THREAD

**Discussion by panziyi1** [pyvista/pyvista#3353] [Q&A] [answered]
I was trying to make [documentation](https://docs.readthedocs.io/en/stable/guides/jupyter.html?highlight=pyvista#rendering-interactive-widgets) with interactive widgets using PyVista.

However, following the [instructions](https://docs.pyvista.org/user-guide/jupyter/pythreejs.html?highlight=jupyter_sphinx#create-interactive-documentation) **the interactive widgets didn't show** on the built webpage:
![1663833878(1)](https://user-images.githubusercontent.com/107454683/191692254-ead904f0-1416-46bc-8300-5c6b6bf84aa7.png)

I thought something is missing between readthedocs and PyVista. Any help would be warmly welcomed. Tks!

My _conf.py_:
![1663834007(1)](https://user-images.githubusercontent.com/107454683/191692730-b3c052ca-cb78-4f5a-beb2-1c9a98b26b36.png)
My _.rst_:
![1663834081(1)](https://user-images.githubusercontent.com/107454683/191693005-9f50d12e-189c-40b1-a793-32508384a98e.png)
My _requirements.txt_：
![1663834050(1)](https://user-images.githubusercontent.com/107454683/191692938-a8dd3e9c-7449-4386-8892-06bdecbb605e.png)

**Comment by banesullivan** [comment]
I think you need to use the `.. pyvista-plot::` sphinx directive from the `pyvista.ext.plot_directive` you are using in your `extensions` rather than just `.. jupyter-execute::`

**Reply by panziyi1** [reply]
Thanks for reply. 
Is `.. pyvista-plot::` designed only for non-interative plots? Can it plot interative widgets? Would you please give an example?
> I think you need to use the `.. pyvista-plot::` sphinx directive from the `pyvista.ext.plot_directive` you are using in your `extensions` rather than just `.. jupyter-execute::`

**Comment by panziyi1** [comment, marked as answer]
It is solved after downgrading ipywidgets to `ipywidgets==7.7.2`.
